### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by WesleyCharlesBlake**
[PR Process](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-process) - [PR Review Checklist](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-review-checklist)

## Description of Changes

Adds the staging k8s environment to a autopublishing branch on netflify called staging.

Staging URL: https://staging--poet-explorer.netlify.com/

Build Logs: https://app.netlify.com/sites/poet-explorer/deploys/5ba1faa3c6aed624823ea032
### Warning

Merging to `master` automatically triggers a deploy in production.

### CODE REVIEW

**Comment by WesleyCharlesBlake** [comment]
@wzalazar @lautarodragan : I tried to replicate what I did on frost-web, but I see my builds are failing for the staging branch and Im not sure what I've missed. Can you help me here please?

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "build:development": "POET_ENV=development webpack",
     "build:testing": "POET_ENV=testing webpack",
     "build:preview": "stylelint './src/**/*.scss' && tslint -p ./tsconfig.json && npm run build:testing",
+    "build:staging": "POET_ENV=staging webpack --mode=production",
```
Comment (wzalazar) [inline package.json]: I think you have to remove `--mode=production`, because explorer-web doesn't have the `webpack v4` yet

Comment (WesleyCharlesBlake) [reply package.json]: I've removed that. But still failing

**Comment by wzalazar** [comment]
@WesleyCharlesBlake you need to create a file in `env/staging.json`

https://github.com/poetapp/explorer-web/blob/6e971f48ce780a2b9d3de716a5a5846e497d0df9/webpack.config.js#L21

**Comment by WesleyCharlesBlake** [comment]
@wzalazar @lautarodragan this is ready to merge now :)

**Review by wzalazar — APPROVED** [review]
* [n/a] Tested changes manually
* [x] Checked accidental architectural/style changes
* [x] Reviewed entire diff
* [n/a] Unit tests
* [n/a] Documentation
* [n/a] Filenames and locations

**Review by kennylavender — APPROVED** [review]
* [skipped] Tested changes manually
* [n/a] Checked accidental architectural/style changes
* [x] Reviewed entire diff
* [n/a] Unit tests
* [n/a] Documentation
* [x] Filenames and locations

This looks good to me. But I don't know anything about the deployment requirements/goals/issues. So I've been leaving these for @lautarodragan. Probably better if he still reviews it.

**Comment by lautarodragan** [comment]
This and the frost PR will require a bit of tweaking. Let’s discuss on the side

**Review by lautarodragan — APPROVED** [review]
* [x] Tested changes manually
* [n/a] Checked accidental architectural/style changes
* [x] Reviewed entire diff
* [n/a] Unit tests
* [n/a] Documentation
* [x] Filenames and locations

Did some super basic manual testing.

**Pull request merged**